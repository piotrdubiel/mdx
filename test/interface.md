# Some Interface

```ocaml
open Cmdliner

val cmd: int Term.t * Term.info
```