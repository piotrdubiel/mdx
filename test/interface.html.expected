<section id="some-interface" class="level1">
<h1>Some Interface</h1>
<div class="highlight">
<pre><code class="language-reason">open Cmdliner;

let cmd: (Term.t(int), Term.info);
</code></pre>
</div>
</section>
